modules/administration/pages/auth-methods-pam.adoc

```diff
@@ -16,19 +16,29 @@ PAM is a suite of libraries that allows you to integrate {productname} with a ce
 In usernames, additionally to alphanumeric characters, [literal]``-``, [literal]``_``, [literal]``.``, and [literal]``@`` are allowed.
 ====
 . Check the [guimenu]``Pluggable Authentication Modules (PAM)`` checkbox.
-. Configure SSSD in the container.
-  Connect inside the container using:
+. Configure SSSD in the server container.
+  At the command prompt of the {productname} container host, as root, enter the server container:
 +
+
 ----
 mgrctl term
 ----
-Then change the file
+
+. Inside the container, execute the following steps:
++
+
+.. To set configuration options, open the file:
 +
 ----
 /etc/sssd/sssd.conf
 ----
 +
-. Set [literal]``krb5_keytab`` in ``domain/$domain`` to [literal]``/etc/sssd/krb5.keytab``
+.. Set [literal]``krb5_keytab`` in ``domain/$domain`` to [literal]``/etc/sssd/krb5.keytab``
```

Comment:
Given the example below I think this part better be removed and replaced by something more generic. It also sends mixed messages where example has different keytab path.

```
Inside the container edit `/etc/sssd/sssd.conf` according to your configuration. 
See example below.
```